source/common/config/config_provider_impl.cc

```diff
@@ -3,29 +3,51 @@
 namespace Envoy {
 namespace Config {
 
-ImmutableConfigProviderImplBase::ImmutableConfigProviderImplBase(
+ImmutableConfigProviderBase::ImmutableConfigProviderBase(
```

Comment:
This file only has 92.9% test coverage, see https://210877-65214191-gh.circle-artifacts.com/0/coverage/coverage.source_common_config_config_provider_impl.cc.html.

Reply:
How did you find that URL?

Reply:
Added test coverage. Will check report once it's done.

Reply:
@fredlas coverage CI job, "Artifacts" tab. You need to be logged into CircleCI with your GH ID for that to work.

Reply:
@AndresGuedez coverage looks good now, thanks.